### PR TITLE
Add member docs to Tensor library

### PR DESCRIPTION
**Pull request by ericstj**

### CODE REVIEW

File: src/System.Numerics.Tensors/System/Numerics/ArrayTensorExtensions.cs [hunk 1]
```diff
@@ -6,144 +6,141 @@ namespace System.Numerics
...
-        /// Creates a copy of this two-dimensional array as a DenseTensor<T>
+        /// Creates a copy of this two-dimensional array as a DenseTensorr&lt;T&gt;
         /// </summary>
-        /// <typeparam name="T"></typeparam>
-        /// <param name="array"></param>
-        /// <param name="reverseStride"></param>
-        /// <returns></returns>
+        /// <typeparam name="T">Type contained in the array to copy to the DenseTensor&lt;T&gt;.</typeparam>
+        /// <param name="array">The array to create a DenseTensor&lt;T&gt; from.</param>
+        /// <param name="reverseStride">False (default) to indicate that the first dimension is most minor (closest) and the last dimension is most major (furthest): row-major.  True to indicate that the last dimension is most minor (closest together) and the first dimension is most major (furthest apart): column-major.</param>
```
Comment (eerhardt) [inline src/System.Numerics.Tensors/System/Numerics/ArrayTensorExtensions.cs]: A thing I found confusing about this description is that it says the default is "first dimension is most minor" and is "row major".  Given that the rows are the first dimension - "row major" would mean that the first dimension is the "most major", right?

Comment (ericstj) [reply src/System.Numerics.Tensors/System/Numerics/ArrayTensorExtensions.cs]: Typo when I was rewording this a few times.  Will fix.

File: src/System.Numerics.Tensors/System/Numerics/ArrayTensorExtensions.cs [hunk 2]
```diff
@@ -6,144 +6,141 @@ namespace System.Numerics
...
-        /// Creates a copy of this three-dimensional array as a DenseTensor<T>
+        /// Creates a copy of this three-dimensional array as a DenseTensorr&lt;T&gt;
         /// </summary>
-        /// <typeparam name="T"></typeparam>
-        /// <param name="array"></param>
-        /// <param name="reverseStride"></param>
-        /// <returns></returns>
+        /// <typeparam name="T">Type contained in the array to copy to the DenseTensor&lt;T&gt;.</typeparam>
+        /// <param name="array">The array to create a DenseTensor&lt;T&gt; from.</param>
+        /// <param name="reverseStride">False (default) to indicate that the first dimension is most major (furthest apart) and the last dimension is most minor (closest together): akin to row-major in a rank-2 tensor.  True to indicate that the last dimension is most major (furtheset apart) and the first dimension is most minor (closest together): akin to column-major in a rank-2 tensor.</param>
```
Comment (eerhardt) [inline src/System.Numerics.Tensors/System/Numerics/ArrayTensorExtensions.cs]: type-o `furtheset`.  😉

Comment (eerhardt) [reply src/System.Numerics.Tensors/System/Numerics/ArrayTensorExtensions.cs]: I think `major (furthest apart)` and `minor (closest together)` are backwards, aren't they?

From wikipedia:  `In a row-major order, the consecutive elements of a row reside next to each other`

Comment (ericstj) [reply src/System.Numerics.Tensors/System/Numerics/ArrayTensorExtensions.cs]: No.  The rows themselves are furthest apart, the columns (AKA: consecutive elements of a row) are closest.

Comment (eerhardt) [reply src/System.Numerics.Tensors/System/Numerics/ArrayTensorExtensions.cs]: But the column itself is spread out - it isn't contiguous.  The row is the one that is contiguous (close together).

Comment (eerhardt) [reply src/System.Numerics.Tensors/System/Numerics/ArrayTensorExtensions.cs]: I guess I see what you are saying.  It's really hard to explain these things in a single sentence.

**Pull request merged**